src/olympia/addons/models.py

```diff
@@ -1604,7 +1604,11 @@ def should_show_permissions(self, version=None):
             self.type == amo.ADDON_EXTENSION
             and version
             and version.file
-            and (version.file.permissions or version.file.optional_permissions)
+            and (
```

Comment:
This entire method appears to be unused, so you can just remove it instead. (If it was used, we'd likely not want to add `host_permissions` support to it yet, since we're not displaying those permissions anywhere at the moment)